metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/request/SearchRequestHandler.java

```diff
@@ -81,11 +82,11 @@ public class SearchRequestHandler {
           .setSkipCache(false)
           .setSkipAggregates(false)
           .setSkipHighlighting(false);
-
   private static final Map<List<EntitySpec>, SearchRequestHandler> REQUEST_HANDLER_BY_ENTITY_NAME = new ConcurrentHashMap<>();
   private static final String REMOVED = "removed";
-
   private static final String URN_FILTER = "urn";
+  private static final String[] FIELDS_TO_FETCH = new String[]{"urn", "usageCountLast30Days"};
```

Comment:
Where are these used? FIELDS_TO_FETCH and URN_FIELD seems like they are unused?

Reply:
I didn't change anything here.. Just moved the variables up one line